packages/mobile/src/store/store.ts

```diff
@@ -57,7 +58,8 @@ const createRootReducer = () =>
     remoteConfig,
     search,
     signon,
-    web
+    web,
+    ...commonReducers()
```

Comment:
👀

Reply:
yeahh... so it technically works, obvs nothing gets updated since sagas aren't live though. do we want to include this big change now you think? or slowly add common reducers?

Reply:
I think it's fine to have all of them now